cli/fathom_web/extract.py

```diff
@@ -36,10 +36,11 @@
 def main(in_directory, preserve_originals):
     """
     Extract resources from the HTML pages in IN_DIRECTORY and store the
-    resources for each page in a newly created directory in IN_DIRECTORY.
+    resources for each page in a newly created page-specific directory
+    within a newly created resources directory in IN_DIRECTORY.
     For example, the resources for `example.html` would be stored in
-    `example_resources/`. This tool is used to prepare your samples for a
-    git-LFS enabled repository.
+    `resources/example_resources/`. This tool is used to prepare your
```

Comment:
Turns out you need 2 backticks on either side in ReStructured Text!

Reply:
Ah yes. Thank you for your great attention to detail!